src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -72,6 +72,9 @@ const {
   Deletion,
 } = ReactTypeOfSideEffect;
 
+const internalErrorMessage =
+  'This error is likely caused by a bug in React. Please file an issue.';
```

Comment:
This will count against the byte size, is this intentional?
It doesn’t matter that much in the grand scheme of things, just wondering if this was intended or not.
If you just hardcoded it everywhere then it would get stripped out.